### PR TITLE
Add some comments and light edits.

### PR DESCRIPTION
**Pull request by codygunton**

### CODE REVIEW

File: cryptography-engineer/src/ec_fft/README.md [hunk 1]
```diff
@@ -4,18 +4,16 @@ Hi there! Welcome to the `ec_fft` test that you are about to take. We will guide
...
+Universal zk-SNARKs like PlonK need to run a one-time trusted setup ceremony to generate a Structured Reference String (SRS). Any number of participants can participate in this ceremony but only _one_ of all the participants needs to be _honest_. As a part of this ceremony, each participant contributes to the setup with their own _secret_ which they are free to choose. If even one of the participants generates this secret randomly and destroys it successfully, the setup ceremony is considered to be successful. Fur our purposes, the output of the setup ceremony is a structure reference string of the form:
 
 $$
 \begin{aligned}
 \mathbb{G}_1 \text{ points: }
 \big([1]_1,[x]_1, [x^2]_1, [x^3]_1, \dots, [x^{N-1}]_1\big), \\
-\mathbb{G}_2 \text{ points: }
```
Comment (codygunton) [inline cryptography-engineer/src/ec_fft/README.md:13]: No benefit to having this here if it's not used IMO.

File: cryptography-engineer/src/ec_fft/README.md [hunk 2]
```diff
@@ -4,18 +4,16 @@ Hi there! Welcome to the `ec_fft` test that you are about to take. We will guide
...
-\big([1]_2,[x]_2, [x^2]_2, [x^3]_2, \dots, [x^{M-1}]_2\big).
 \end{aligned}
 $$
 
-Here, $x \in \mathbb{F}$ is the combined secret of all the participants and is assumed to be unknown to anyone in the world. Further, the notation $[a]_1 := aG_1$ such that $G_1\in \mathbb{G}_1$ is generator for the first group $\mathbb{G}_1$.
+Here, $\mathbb{G}_1$ is a cyclic group, $\mathbb{F}$ is a finite field, and the order of $\mathbb{G}_1$ is the same as the order of $\mathbb{F}$. The element  $x \in \mathbb{F}$ is the combined secret of all the participants and is assumed to be unknown to anyone in the world. Further, we define $[a]_1 := aG_1$, where $G_1\in \mathbb{G}_1$ is a fixed generator $\mathbb{G}_1$.
```
Comment (codygunton) [inline cryptography-engineer/src/ec_fft/README.md:16]: Whoops, should say a "fixed generator *of*..."

File: cryptography-engineer/src/ec_fft/README.md [hunk 3]
```diff
@@ -41,11 +39,11 @@ L_{n,i}(\omega^j) =
 \end{cases}.
 $$
 
-In other words, the Lagrange basis polynomial $L_{n,i}(X)$ is $1$ on $\omega^i$ and $0$ on the other roots $\{\omega^j\}_{j \neq i}$. It is sometimes useful to work with the Lagrange form of a polynomial than its coefficient form.
+In other words, the Lagrange basis polynomial $L_{n,i}(X)$ is $1$ on $\omega^i$ and $0$ on the other roots $\{\omega^j\}_{j \neq i}$. The Lagrange form of a polynomial is sometimes more useful than the monomial form.
 
 #### Fast Fourier Transform
 
-Given the coefficent form of a polynomial, it is possible to convert it to the Lagrange form using the Discrete Fourier Transform (DFT) operation. Similarly, we can take an inverse DFT to convert the Lagrange form to its coefficient form.
+Given the coefficent form of a polynomial, it is possible to convert it to the Lagrange form using the Fast Fourier Transform (FFT) operation. Similarly, we can take an inverse FFT to convert the Lagrange form to its coefficient form.
```
Comment (codygunton) [inline cryptography-engineer/src/ec_fft/README.md:46]: Why call it DFT if you use the notation FFT for the name of the map?

**Review by codygunton — COMMENTED** [review]
IMO it's a little unclear what the objective is after this. E.g., we're going to accomplish this goal without knowing $x$?

**Comment by suyash67** [comment]
> IMO it's a little unclear what the objective is after this. E.g., we're going to accomplish this goal without knowing x?

ECFFT is useful in danksharding and protocols like CQ (i.e. for precomputing cached quotients). But I didn't want to make this exercise too difficult by including the Fiest-Khovratovich computation (i.e. multi-kzg proofs) so I just included EC FFT as a primitive.

File: cryptography-engineer/src/ec_fft/README.md (hunk 2 above)
Comment (suyash67) [reply cryptography-engineer/src/ec_fft/README.md:16]: Makes sense, thanks!

File: cryptography-engineer/src/ec_fft/README.md (hunk 1 above)
Comment (suyash67) [reply cryptography-engineer/src/ec_fft/README.md:13]: Right

**Pull request merged**